src/friendly-iframe-embed.js

```diff
@@ -857,7 +857,11 @@ export class FriendlyIframeEmbed {
 function installPolyfillsInChildWindow(parentWin, childWin) {
   installDocContains(childWin);
   installDOMTokenList(childWin);
-  if (isExperimentOn(parentWin, 'custom-elements-v1') || getMode().test) {
+  if (
+    // eslint-disable-next-line no-undef
```

Comment:
Do you think we should continue to disable via ESLint these lines or should we fix the issue so ESLint can do the right thing by default here. (Example always use a value from a known Object)

Reply:
I can't remember how to declare a global in eslint anymore…